antarest/study/service.py

```diff
@@ -887,7 +888,7 @@ def download_outputs(
         output_id: str,
         data: StudyDownloadDTO,
         params: RequestParameters,
-    ) -> MatrixAggregationResult:
+    ) -> Union[MatrixAggregationResult, FileDownloadTaskDTO]:
```

Comment:
this method should take two additional parameters, the type of export (currently in StudyDownloadDTO) and a boolean indicating if it should use the task service or not